src/sentry/static/sentry/app/components/loadingError.tsx

```diff
@@ -36,23 +36,28 @@ class LoadingError extends React.Component<Props> {
   render() {
     const {message, onRetry} = this.props;
     return (
-      <Alert type="error">
+      <StyledAlert type="error">
         <Content>
           <IconInfo size="lg" />
           <div data-test-id="loading-error-message">{message}</div>
           {onRetry && (
-            <Button onClick={onRetry} priority="default" size="small">
+            <Button onClick={onRetry} type="button" priority="default" size="small">
               {t('Retry')}
             </Button>
           )}
         </Content>
-      </Alert>
+      </StyledAlert>
     );
   }
 }
 
 export default LoadingError;
 
+const StyledAlert = styled(Alert)`
```

Comment:
It's possible this is rendered outside of a `Panel` which will cause it to look out of place